scripts/scan-all.sh

```diff
@@ -62,6 +62,7 @@ plugin_map["quay.io/fairwinds/insights-utils"]="utils"
 plugin_map["quay.io/fairwinds/workloads"]="workloads"
 plugin_map["quay.io/fairwinds/on-demand-job-runner"]="on-demand-job-runner"
 plugin_map["quay.io/fairwinds/kyverno-policy-sync"]="kyverno-policy-sync"
+plugin_map["quay.io/fairwinds/insights-event-watcher"]="watcher"
```

Comment:
can we use 
```
plugin_map["quay.io/fairwinds/event-watcher"]="event-watcher"
```
?

Reply:
quay name was already defined by Andy, it is ready